ush/ioda/bufr2ioda/bufr2ioda_satwind_scat.py

```diff
@@ -0,0 +1,369 @@
+#!/usr/bin/env python3
+import argparse
+import numpy as np
+import numpy.ma as ma
+from pyiodaconv import bufr
+import calendar
+import json
+import time
+import math
+import datetime
+import os
+from datetime import datetime
+from pyioda import ioda_obs_space as ioda_ospace
+from wxflow import Logger
+
+# ==============================================================================================
+# Subset    |  Description                                              |  PrepBUFR Report Type
+# ----------------------------------------------------------------------------------------------
+# NC012122  |  Level-2 ocean surface wind vector retrievals and         |  290
+#           |  derived wind components from the Advanced Scatterometer  |
+#           |  (ASCAT) on MetOp satellites at 50-km sampling resolution |
+# ==============================================================================================
+
+
+def Compute_WindComponents_from_WindDirection_and_WindSpeed(wdir, wspd):
```

Comment:
No need to do now, but we should pull this function out somewhere since it is duplicated/shared among wind observations.

Reply:
@CoryMartin-NOAA Can not agree with you more.  Ron and I had talked about this.  We should have a common Python library for common shared functions.

Reply:
We are not sure where is the best place for this common shared library for Python functions. 
Any thoughts?

Reply:
Depending on how general this would be, it could be its own python package/stand alone repository. Originally, this was one of the concepts of EMCPy, but now, I'm not sure. Perhaps it still has its place there?